sympy/geometry/parabola.py

```diff
@@ -376,6 +376,29 @@ def p_parameter(self):
             p = sign(self.focus.args[1] + y)
         return p * self.focal_length
 
+    @property
+    def latus_rectum(self):
+        """The latus rectum of a conic section is the chord through
+           a focus parallel to the conic section directrix (Coxeter 1969).
+           "Latus rectum" is a compound of the Latin latus, meaning "side," and rectum, meaning "straight."
+
+           Returns
+           =======
+
+           latus_rectum : Line
+               Latus Rectum of Parabola
+
+           Examples
+           ========
+
+           >>> from sympy import Point, Line, Parabola
+           >>> p1 = Parabola(Point(0, 0), Line(Point(5, 8), Point(7, 8)))
+           >>> p1.latus_rectum
+           Line2D(Point2D(0, 0), Point2D(1, 0))
```

Comment:
This is not a chord, it's a line. It should return a Segment whose endpoints are where this line intersects the Parabola.